cleanlab/rank.py

```diff
@@ -584,6 +585,9 @@ def get_knn_distance_ood_scores(
     avg_nbrs_distances : np.array
       Average distance to k-nearest neighbors for each datapoint which is used as a score for OOD detection.
     """
+    # if nbrs is not provided, then use default KNN classifier (k=10)
+    if nbrs is None:
+        nbrs = NearestNeighbors(n_neighbors=10, metric="cosine").fit(features)
```

Comment:
> Should we create default NN with 10 neighbors always or 'k' if k exists?

Let's do k if k exists, but 10 as the default otherwise if k = None and nbrs not given.